src/geo/ui/widgets/category/model.js

```diff
@@ -86,6 +86,19 @@ module.exports = WidgetModel.extend({
       });
     }, this);
 
+    this.bind('change:collapsed', function(mdl, isCollapsed) {
```

Comment:
This is also duplicated [here](https://github.com/CartoDB/cartodb.js/pull/883/files#diff-214e2a6e2deb1d445b62d408f340b33eR55). Do we need it here? Just double checking...